#291 update header and footer, update community page, add ecosystem, usage and history page (#303)
OpenRefine/openrefine.org: src/pages/openrefine_history.md

---
title: OpenRefine Project History
description: A brief overview of OpenRefine History
hide_table_of_contents: false
---
# OpenRefine History 

> Over the last ten years, OpenRefine has undergone significant transformations, including changes in its name and development model. It has transitioned from an enterprise-led model to a grant-based funding model.

Reviewer: I agree with Tom's suggestion, but I think it's still worth mentioning grants because they do have a big impact on what gets worked on (I would say). Perhaps something like this? "It has transitioned from an enterprise-led open source model to a community-based open source model, with later support from occasional grants."

> Over the last ten years, OpenRefine has undergone significant transformations, including changes in its name and development model. It has transitioned from an enterprise-led model to a grant-based funding model.

Author: I updated the text accordingly.

```mermaid
%%{init: { 'theme': 'forest', 'timeline': {'disableMulticolor': true}}}%%
timeline
    title OpenRefine timeline
    . : Project Name
                        : Main Release
                        : Development Model

      2008 - 2010 : Freebase Gridworks
                            : Version 1.0
                            : Lead by Metaweb Technologies, Inc. with open source contributors

    2010 - 2012 : Google Refine
                    : Version 2.0 and 2.5
                    : Lead by Google with open source contributors

    2012 - 2018 : OpenRefine
                    : Version 2.6

>                     : Solely with volunteer open source contributors

Author: I am interested in showing the evolution of the **governance and sustainability model**, but not so much the funding source. On the `Development model` line, I want to represent the change from enterprise-led to the 2022-2018 period (for which I lack good words to describe it) to a foundation-like model with CS&S.

      2018 - 2019 : OpenRefine
                    : Version 3.0
                    : First grants with Google News Initiatives and CZI with open source contributors

      2020 - Today : OpenRefine

>                     : Version 4.x and 3.7.x

Author: I am keen to keep a mention of OpenRefine 4.x. I am leaving this comment open for other to chime in.

>                     : Version 4.x and 3.7.x

Reviewer: 3.3+ is simpler, it's one less thing to update when we make new releases.

```

## 2008 - 2010: Freebase Gridworks
<img src="/img/logo/freebase.png" alt="Freebase Gridworks" height="50"/>
The history of OpenRefine dates back to May 2010, when Metaweb Technologies, Inc. initiated the development of an open-source data cleansing tool named Freebase Gridworks. Freebase Gridworks, conceived by David Huynh, simplified data cleaning and transformation data to import into Freebase, a large collaborative knowledge base built by Metaweb. Freebase Gridworks offered a user interface for editing and reconciling large datasets.

## 2010 - 2012: Google Refine
<img src="/img/logo/google-refine.png" alt="Google Refine" height="50"/>
In July 2010, Google acquired Metaweb Technologies and with it, Freebase Gridworks. Following this acquisition, the tool was renamed "Google Refine". Under Google's stewardship, Google Refine 2.0 was released, featuring enhanced data cleaning capabilities. This version established the tool's reputation for handling various data-related tasks, such as cleaning messy data, transforming data formats, and extending data sets with web services.

## 2012 - 2018: OpenRefine
<img src="/img/logo/open-refine.png" alt="OpenRefine" height="50"/>

> In 2012, Google made the decision to discontinue the development of Google Refine. Existing project contributors took over the project and renamed it "OpenRefine". The project was then moved to Github. The following year saw OpenRefine gaining a diverse user base, including scientists and researchers, journalists, semantic web practitioners, librarians, and cultural heritage specialists.

Author: thanks for catching this. I didn't meant to refer a change in the user base, I only wanted to list the different user community in this section.

> In 2017, OpenRefine began integrating with Wikidata by creating a reconciliation endpoint and a process to publish data to the Wikimedia project.

Reviewer: In a way, phrased like this it gives the impression that the Wikidata recon service is made by the OpenRefine project, and we've tried to keep the two things separate, so perhaps we could rather phrase it like this:
```suggestion
In 2017, OpenRefine began integrating with Wikidata thanks to the availability of a reconciliation endpoint and a process to publish data to the Wikimedia project.
```

> ## 2018 - today: OpenRefine grant base model

Reviewer: The phrase "grant base model" doesn't feel very natural to me.
Although the Google News initiative funding is indeed the first grant of a series, I would rather make this phase start later and correspond to joining CS&S. That's a governance change that feels more significant for the project.
So it could be:
```suggestion
## 2019 - today: OpenRefine, part of Code for Science & Society
```
(not sure if you want to use 2019 or 2020 as date, fine either way by me).

<img src="/img/logo/OpenRefine.png" alt="OpenRefine" height="50"/>
In 2018, Google News Initiative donated $100,000 to support OpenRefine development, initiating a new grant-based sustainability model. The same year the project's logo was updated.

OpenRefine confirmed its grant-based sustainability model the following year when it received USD 200,000 through the Chan-Zuckerberg Initiative EOSS-1. In 2020, OpenRefine became a fiscally sponsored project of Code for Science and Society, a 501(c)(3) charitable organization in the USA, and an Advisory Committee was formed. During that time, OpenRefine experienced tremendous growth by doubling the number of active contributors, increasing the number of translated languages, and seeing more users rely on it.

Since then, OpenRefine has secured recurring funding from CZI, the Wikimedia Foundation, and NFDI to ensure the project's long-term viability. This funding has helped the project rely on a combination of volunteers and contractors for its development.

> 

Author: Several users I spoke with last year mentioned that the lack of information on OpenRefine's sustainability model made it harder for their organization to adopt OpenRefine. The funding is one aspect of it, as the number of contributors, which is also mentioned in this section.

> 

Reviewer: I think both would be useful. We could integrate brief summaries of the main improvements to OpenRefine in the current text.

> 

Author: I tried to rewrite the section based on both feedback.
